index.src.html

```diff
@@ -2459,7 +2459,7 @@
 
   : <dfn>Relying Party</dfn>
   : <dfn>WebAuthn Relying Party</dfn>
-  :: The entity which needs to rely in the authentication provided by the WebAuthn specification. At the conclusion of the registration, the Relying Party has the public key that was created by the Authenticator.
+  :: The entity which needs to rely in the authentication provided by the WebAuthn specification. When [[#registration-embedded]] concludes, the Relying Party has the public key that was created by the Authenticator.
```

Comment:
Apologies for the noise, but I take this back. It might be better not to have the link here. This is how it now renders after Bikeshed processing:

When §1.1 Registration (embedded authenticator mode) concludes, the Relying Party has the public key that was created by the Authenticator.

May be better just to have the word "registration" instead of the link.